### PR TITLE
Add method to cancel process

### PR DESCRIPTION
**Pull request by theKhorshed**
Thanks for the awesome works!

I was in need to let my user cancel the process if they want. So, I think it would be better if it's available in the plugin itself.

Thanks again.

### CODE REVIEW

File: classes/wp-background-process.php [hunk 1]
```diff
@@ -441,6 +441,24 @@ protected function clear_scheduled_event() {
...
 
+        /**
+         * Cancel Process
+         *
+         * Stop processing rest queue items, clear cronjob and delete batch.
+         *
+         */
+        public function cancel_process() {
+
+          if ( $this->is_queue_empty() ) {
```
Comment (A5hleyRich) [inline classes/wp-background-process.php]: Should this be `if ( ! $this->is_queue_empty() )`?

**Comment by A5hleyRich** [comment]
Thanks for the PR. Just one comment.

**Comment by theKhorshed** [comment]
Yes! Updated

**Comment by A5hleyRich** [comment]
Thanks. PR merged.

**Pull request merged**